tests/integration/internal/constants.go

```diff
@@ -13,12 +13,80 @@ const (
 
 	EnvNameKindImage = "KIND_NODE_IMAGE"
 
-	YamlPathReceiverMock = "yamls/receiver-mock.yaml"
+	YamlPathReceiverMock    = "yamls/receiver-mock.yaml"
+	ReceiverMockServicePort = 3000
 )
 
+// metrics we expect the receiver to get
+// metrics which are expected in current E2E tests but aren't present here, are commented out
+// TODO: figure out why the expected metrics aren't present
 var (
-	HelmSumoLogicChartAbsPath string
-	KindImages                KindImagesSpec
+	KubeStateMetrics = []string{
+		"kube_statefulset_status_observed_generation",
+		"kube_statefulset_status_replicas",
+		"kube_statefulset_replicas",
+		"kube_statefulset_metadata_generation",
+	}
+	KubeDaemonSetMetrics = []string{
+		"kube_daemonset_status_current_number_scheduled",
+		"kube_daemonset_status_desired_number_scheduled",
+		"kube_daemonset_status_number_misscheduled",
+		"kube_daemonset_status_number_unavailable",
+	}
+	KubeDeploymentMetrics = []string{
+		"kube_deployment_status_replicas_available",
+		"kube_deployment_status_replicas_unavailable",
+		"kube_deployment_spec_replicas",
+	}
+	KubeNodeMetrics = []string{
+		"kube_node_info",
+		"kube_node_status_allocatable",
+		"kube_node_status_capacity",
+		"kube_node_status_condition",
+	}
+	KubePodMetrics = []string{
+		"kube_pod_container_info",
+		"kube_pod_container_resource_requests",
+		"kube_pod_container_resource_limits",
+		"kube_pod_container_status_ready",
+		"kube_pod_container_status_terminated_reason",
+		"kube_pod_container_status_restarts_total",
+		"kube_pod_status_phase",
+	}
+	KubeletMetrics = []string{
+		"kubelet_running_containers",
+		"kubelet_running_pods",
+	}
+	CAdvisorMetrics = []string{
+		"container_cpu_usage_seconds_total",
+		// "container_fs_usage_bytes",
+		// "container_fs_limit_bytes",
```

Comment:
I believe this one was is related to https://github.com/google/cadvisor/pull/2872

Reply:
So it'll start working after kind gets to containerd 1.5.7: https://github.com/containerd/containerd/issues/5882? I'll add a comment.